api-reference/receipts/endpoints.markdown

```diff
@@ -0,0 +1,672 @@
+---
+title: Endpoints
+layout: reference
+---
+
+# Endpoints
+
+- [Definitions of Resources](#definitions-of-resources)
+- [General](#general)
+  - [GET Service Index](#endpoint-service-index)
+  - [GET Receipt Status by Receipt ID](#endpoint-get-receipt-status)
+- [eReceipts](#ereceipts)
+  - [GET Schemas](#endpoint-schemas)
+  - [POST Receipts](#endpoint-post-a-receipt)
+  - [GET Receipts by User ID](#endpoint-get-receipts-by-userid)
+  - [GET Receipts by Receipt ID](#endpoint-get-a-receipt-by-id)
+  - [GET Receipt Image by Receipt ID](#endpoint-get-receipt-image)
+- [Image-Only Receipts](#image-only-receipts)
+  - [POST Image-Only Receipts](#endpoint-post-an-image-only-receipt)
+  - [GET Image-Only Receipts by User ID](#endpoint-get-image-only-receipts-by-userid)
+  - [GET Image-Only Receipt by Receipt ID](#endpoint-get-an-image-only-receipt-by-id)
+  - [GET Receipt Image by Receipt ID](#endpoint-get-receipt-image-image-only)
+
+#### Definitions of Resources
```

Comment:
This should be a level-3 header, like General, eReceipts, etc. (three hashes)

Reply:
👍